utils/paramfetch/src/lib.rs

```diff
@@ -117,18 +128,26 @@ pub async fn get_params(
 /// Get proofs parameters and all verification keys for a given sector size using default manifest.
 #[inline]
 pub async fn get_params_default(
+    data_dir: PathBuf,
     storage_size: SectorSizeOpt,
     is_verbose: bool,
 ) -> Result<(), anyhow::Error> {
-    get_params(DEFAULT_PARAMETERS, storage_size, is_verbose).await
+    get_params(
+        data_dir.clone(),
+        DEFAULT_PARAMETERS,
+        storage_size,
+        is_verbose,
+    )
+    .await
 }
 
 async fn fetch_verify_params(
+    data_dir: PathBuf,
     name: &str,
     info: Arc<ParameterData>,
     mb: Option<Arc<MultiBar<Stdout>>>,
 ) -> Result<(), anyhow::Error> {
-    let path: PathBuf = [&param_dir(), name].iter().collect();
+    let path: PathBuf = [param_dir(data_dir.clone()).to_str().unwrap(), name].iter().collect();
```

Comment:
Refactor to avoid the `unwrap`. In general, if you use `unwrap`, you have to write a comment with a proof of why it will not panic.